### PR TITLE
[docs] Reposition legacy callout in Push notifications setup

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Context [slack](https://exponent-internal.slack.com/archives/C064NCHAT4P/p1702906857207119?thread_ts=1702906196.548389&cid=C064NCHAT4P)

# How

Move the callout at the top of the page so it isn't overlooked.

# Test Plan

N/A

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/push-notifications/push-notifications-setup.mdx [hunk 1]
```diff
@@ -8,6 +8,8 @@ import { Terminal } from '~/ui/components/Snippet';
 import { Step } from '~/ui/components/Step';
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 
+> Expo Notifications only supports the **Cloud Messaging API (Legacy)** key at this time. This key is deprecated by Firebase. However, it will continue to work until June 20, 2024. We will provide information on migrating to the new v1 key in the future.
```
Comment (kadikraman) [inline docs/pages/push-notifications/push-notifications-setup.mdx:11]: Should we mention we'd anticipate this to happen in Q1 next year?

**Review by Simek — APPROVED** [review]
👌

File: docs/pages/push-notifications/push-notifications-setup.mdx (hunk 1 above)
Comment (amandeepmittal) [reply docs/pages/push-notifications/push-notifications-setup.mdx:11]: I'm not sure about that. We don't have a specific date set so I'd avoid it but don't have a personal opinion here.

**Review by kadikraman — APPROVED** [review]
🚀

**Pull request merged**